update runtime service updates section (#2808)
Qiskit/documentation: docs/guides/latest-updates.mdx

{/* use url format /api/qiskit/release-notes/X.X to link to the specific X.X version of the release notes */}

### Qiskit Runtime client 0.37.0

*To see the release notes for all versions, visit the [Qiskit Runtime client release notes](/api/qiskit-ibm-runtime/release-notes).*

**Highlights from the 0.37.0 (13 March 2025) version release**

#### Deprecation

> The `IBMBackend` attributes `max_shots` and `max_experiments`, as well as the property `max_circuits` have all been deprecated. These attribute values used to represent the maximum number of shots and circuits that could be submitted in one job. See [job limits](/guides/job-limits#job-limits) for details.

Reviewer: > These attribute values used to represent the maximum number of shots and circuits that could be submitted in one job.

Something like this, to remove ambiguity around "used" :
"These attribute values were used to represent" / "...had been used to represent" / "Formerly, these attributes represented..."

> The `IBMBackend` attributes `max_shots` and `max_experiments`, as well as the property `max_circuits` have all been deprecated. These attribute values used to represent the maximum number of shots and circuits that could be submitted in one job. See [job limits](/guides/job-limits#job-limits) for details.

Author: ```suggestion
The `IBMBackend` attributes `max_shots` and `max_experiments`, as well as the property `max_circuits` have all been deprecated. Previously, these attribute values represented the maximum number of shots and circuits that could be submitted in one job. See [job limits](/guides/job-limits#job-limits) for details.
```

#### New features

> qiskit-ibm-runtime is now compatible with Qiskit 2.0. This means that classes and methods removed in Qiskit 2.0 have also been removed or handled accordingly.  The `pulse` and `channel` related changes are highlighted below. See the [Pulse migration guide](/migration-guides/pulse-migration) for details.

Reviewer: ```suggestion
qiskit-ibm-runtime is now compatible with Qiskit v2.0. This means that classes and methods removed in Qiskit v2.0 have also been removed or handled accordingly.  The `pulse` and `channel` related changes are highlighted below. See the [Pulse migration guide](/migration-guides/pulse-migration) for details.
```

> qiskit-ibm-runtime is now compatible with Qiskit v2.0. This means that classes and methods removed in Qiskit v2.0 have also been removed or handled accordingly.  The `pulse` and `channel` related changes are highlighted below. See the [Pulse migration guide](/migration-guides/pulse-migration) for details.

Reviewer: ```suggestion
qiskit-ibm-runtime is now compatible with Qiskit v2.0. This means that classes and methods removed in Qiskit v2.0 have also been removed or handled accordingly.  The `pulse` and `channel` related changes are highlighted in the following list. See the [Pulse migration guide](/migration-guides/pulse-migration) for details.
```

This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use.

### 27 February 2025

Early Access release of the [new IBM Quantum Platform](https://quantum.cloud.ibm.com/)

In this release, Open and Pay-As-You-Go users can begin the onboarding experience, update their existing code to use the new IBM Cloud channel, and explore the platform. Refer to the [migration guide](/migration-guides/classic-iqp-to-cloud-iqp) for help.

As an incentive to migrate, all users, including Open Plan, can access an IBM Quantum Heron QPU (IBM Torino) when using the new IBM Quantum Platform.

<Admonition type="note">
Premium Plan, Dedicated service, and Startup program users will receive guided migration support through their administrators in the coming months. In the meantime, they can create a free Open plan account to explore the new platform.
</Admonition>

Read more details in the [blog.](https://www.ibm.com/quantum/blog/iqp-upgrade)

### 2 February 2025

> A new version of [dynamic circuits](/guides/classical-feedforward-and-control-flow) will start rolling out on 2 June 2025. This new version lets you run dynamic circuits at utility scale and comes with improvements in speed, quality, and usability that will gradually be added throughout the year.  Improvements include the following:

Reviewer: ```suggestion
Starting 2 June 2025 and continuing through the year, IBM Quantum will begin a gradual rollout of new features to [dynamic circuits](/guides/classical-feedforward-and-control-flow) that will enable them at the utility scale.
Improvements include the following:
```

Reviewer: (This matches the announcement)

> - Using the Gen3 engine stack that gives a 75x speedup in execution
> - Executing branches in parallel to reduce circuit duration and improves result quality
> - Improving mid circuit measurements
> - Improving integration with dynamical decoupling
> - Improving circuit timing information visibility

Reviewer: ```suggestion
- A 75x speedup in execution by using the [Gen3 engine stack](/guides/runtime-options-overview#experimental-option-note)
- Parallel execution of branches, which reduces circuit duration and improves result quality
- Improvement in mid-circuit measurement
- Better integration with dynamical decoupling
- More visibility of circuit timing information
```

Author: But wouldn't it be better if these were parallel? @abbycross

Reviewer: Aren't they parallel? All nouns? Also, it's how we have it in the announcement - seems less "singsongy" than having the gerunds in every bullet

Author: Yeah; I can see that - always fighting the passive voice, but I can live with it. :)

> To accelerate their roll-out, the following capabilities are now deprecated and will be no longer supported on or around 2 June 2025:
> 
> - The while, for, and switch control flow constructs
> - The ability to use control flow instructions inside the body of a branch (that is, nested control flow)
> - Conditional measurements

Reviewer: ```suggestion
To accelerate the rollout, the following capabilities are now deprecated and will be no longer supported on or around 2 June 2025:

- The `while`, `for`, and `switch` control flow constructs
- The ability to use control flow instructions inside the body of a branch (as in, nested control flow)
- Conditional measurements
```

### 7 January 2025

> The rollout of the Gen3 engine stack begins today. This engine stack gives a 75x speedup in execution.  The rollout will be done on one QPU at a time, where a percentage of the jobs on the QPU will use the Gen3 path.  You can also force the gen3 path using the [`gen3-turbo` experimental option.](/guides/runtime-options-overview#available-options) Note that this engine stack is not compatible with fractional gates, pulse gates, or dynamic circuits.

Reviewer: ```suggestion
The rollout of the Gen3 engine stack begins today. This engine stack gives a 75x speedup in execution.  The rollout will be done on one QPU at a time, where a percentage of the jobs on the QPU will use the Gen3 path.  You can also force the Gen3 path using the [`gen3-turbo` experimental option.](/guides/runtime-options-overview#available-options) Note that this engine stack is not compatible with fractional gates, pulse gates, or dynamic circuits.
```
